prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html

```diff
@@ -1,11 +1,25 @@
+<app-alert *ngIf="licenseClassDiscontinued"
+           type="danger"
+           icon="warning"
+           class="mb-4">
+  <ng-container #alertContent
+                class="alert-content">
+    {{ this.collegeDiscontinued ?
+    "Your previous licence class is no longer available, please " :
+    "The name of your college has changed. Please select the new name from the dropdown list for college name, " +
+    "then "}}
+    use the dropdown menu to select the licence you currently hold.
```

Comment:
This seems reversed, possibly because of how it is set in `checkLicenseIfDiscontinued`?

Reply:
Good catch. The logic and the code have been updated accordingly.